e2e/charts/newrelic-infrastructure/templates/daemonset-kubelet.yaml

```diff
@@ -80,6 +80,8 @@ spec:
           {{- end }}
         - name: agent
           image: {{ .Values.agentImage.repository }}:{{ .Values.agentImage.tag }}
+          command: [ "/sbin/tini" ]
+          args: [ "--", "/usr/bin/newrelic-infra" ]
```

Comment:
Mostly a minor thing, but I would just mash all this together in the `command` line (which is the equivalent of `ENTRYPOINT`) rather than having both `command` and `args`

Reply:
done